src/lib/components/support.svelte

```diff
@@ -40,7 +44,8 @@
             <h4 class="eyebrow-heading-3">Premium support</h4>
             {#if isPaid}
                 <p class="u-line-height-1-5 u-margin-block-start-8">
-                    Get personalized support from the Appwrite team from <b>09:00 - 17:00 UTC</b>
+                    Get personalized support from the Appwrite team from <b>{supportTimings}</b>
+                    <Label tooltip={localeTimezoneName()} />
```

Comment:
It does look like it shows the correct time span for me: 
<img width="330" alt="image" src="https://github.com/appwrite/console/assets/1477010/a95f7cee-6104-41f5-8155-f60cbc7cc576">

Sadly, the tooltip button is pushed to the next line and the veritcal alignment isn't right. I'm going to have someone else try and improve this.

Reply:
We could get the icon on the same line by changing the support text to something like one of the below -
1. Personalized support from the Appwrite team at <b>{supportTimings}</b>
2. Receive personalized support from the Appwrite team, available during <b>{supportTimings}</b>
3. Personalized support by the Appwrite team is available from <b>{supportTimings}</b>

Also, I am not sure of what you mean in the second comment 😅. Do you want me to close this PR?

Reply:
> Also, I am not sure of what you mean in the second comment 😅. Do you want me to close this PR?

I'm going to reach out to a team member, but he's OOO until next week.

Reply:
Alright, no worries.

Reply:
Alright, I spoke with our design team and they said we can just add the long time zone like:

> Get personalized support from the Appwrite team from 02:00 - 10:00 Pacific Time.

Let's make the time zone bold just like the time too.

Reply:
Done.